jaxrs-clients/build.gradle

```diff
@@ -55,7 +55,15 @@ shadowJar {
         exclude(dependency { it.moduleGroup.contains("javax.ws.rs") })
         exclude(dependency { it.moduleGroup.contains("com.fasterxml.jackson") })
     }
-    classifier = ''
+}
+
+publishing.publications {
+    nebulaShadow(MavenPublication) {
+        artifactId project.name + '-all'
```

Comment:
ok, so i'd now have to depend on 'c.p.remoting1:jaxrs-clients-all' if I want the shaded version? sounds ok.